docs/modules/data-structures/pages/vector-collections.adoc

```diff
@@ -0,0 +1,679 @@
+= Vector Collection
+:page-enterprise: true
+:page-beta: true
+
+== Intro
+
```

Comment:
```suggestion

For further information on the architecture and considerations for the Beta version of the Vector Collection data structure, see xref:data-structures:vector-search-overview.adoc[].

```

Reply:
Added a x-ref to the vector design overview in https://github.com/hazelcast/hz-docs/pull/1134there is a comment in that PR explaining the rational (and why the path is now pointing to data-structures instead of architecture)